www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -667,6 +667,11 @@
         <default>0</default>
         <Required>Y</Required>
       </disable_bot_protection>
+      <naxsi_whitelist_srcip type="CSVListField">
+        <Required>N</Required>
+        <mask>/^((?:\d+\.){3,3}\d+|[a-f0-9\:]+)(?:\/\d+)?(,?(?:(?:(\d+\.){3,3}\d+|[a-f0-9\:]+)(?:\/\d+)?))*$/i</mask>
```

Comment:
you may want to support CIDR here.

Reply:
```suggestion
 <mask>/^([\/0-9a-fA-F.:,])*/u</mask>
```

Reply:
* this change breaks the file layout (you remove the indent)
* add the i flag for not case sensitive instead of adding both (performs better)